models/models.go

```diff
@@ -785,22 +795,22 @@ func (m *RouteManager) deployCertificate(route Route, cert acme.CertificateResou
 		"instance-id": route.InstanceId,
 	})
 
+	lsession.Info("get-cert-expiry")
 	expires, err := acme.GetPEMCertExpiration(cert.Certificate)
 	if err != nil {
-		lsession.Error("get-cert-expiry", err)
+		lsession.Error("get-cert-expiry-err", err)
 		return err
 	}
 
 	name := fmt.Sprintf("cdn-route-%s-%s", route.InstanceId, expires.Format("2006-01-02_15-04-05"))
-
-	m.logger.Info("Uploading certificate to IAM", lager.Data{"name": name})
-
+	m.logger.Info("iam-upload-certificate", lager.Data{"name": name})
 	certId, err := m.iam.UploadCertificate(name, cert)
 	if err != nil {
-		lsession.Error("iam-upload-certificate", err)
+		lsession.Error("iam-upload-certificate-err", err)
```

Comment:
And here?

Reply:
Resolved by including the domains in all this function via the `logger.Session` call.